plugins/node.d.linux/diskstat_.in

```diff
@@ -23,6 +23,37 @@ for running as root enter this in a plugin configuration file:
   [diskstat_*]
     user root
 
+You can sym link to this wildcard plugin in various ways:
+
+- diskstat_iops_sda
+- diskstat_latency_nvme2n1
+- diskstat_throughput_nvme0n1
+
+The segments are separated by _ and denote:
+
+- name of the plugin. Always diskstat.
+- What you're measuring: iops (IO/s), latency (included utilization) and
+  throughput.
+- Device: like sda, xvda, nvme0n1 (the latter which, incidentally, still
```

Comment:
Please mention that also `/dev/disk/by-uuid/FOO` is possible here. I think, people will appreciate this detail a lot :)